src/AppBundle/EventListener/ControllerListener.php

```diff
@@ -28,13 +28,22 @@
  */
 class ControllerListener
 {
+    /**
+     * @var SourceCodeExtension
+     */
     protected $twigExtension;
 
+    /**
+     * @param SourceCodeExtension $twigExtension
```

Comment:
this adds exactly no value compared to what is provided by the method signature. In such case, we don't add meaningless phpdoc in Symfony